src/main/java/com/carrotcreative/cream/strategies/generic/CacheStrategy.java

```diff
@@ -6,14 +6,14 @@
 
 import java.io.Serializable;
 
-public interface CacheStrategy<T extends LoaderParams> {
+public interface CacheStrategy<T extends LoaderParams, C extends Serializable> {
```

Comment:
Ah, makes sense.  I would say you're absolutely welcome to go that way for any libs, as I don't have a strong preference.

Just for cream, let's keep using specific names as we've already gone that direction.  If you wanted you change it to single letters I would be fine, just refactor everything to be single letters!